DependencyInjection/SwarrotExtension.php

```diff
@@ -75,12 +70,20 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->setParameter('swarrot.messages_types', $messagesTypes);
 
-        if ($container->getParameter('kernel.debug')) {
+        if ($config['enable_collector']) {
             $loader->load('collector.xml');
         }
     }
 
     /**
+     * {@inheritDoc}
+     */
+    public function getConfiguration(array $configs, ContainerBuilder $container)
```

Comment:
Overwriting this method allows to keep the bundle compatible with ``config:dump-reference``